docs/endpoints/user.md

```diff
@@ -12,33 +14,100 @@ Create a new user.
 ```
 
 #### Body
-The body is a json format of a user.
+The body is a json format of a user, with an embedded reference to the Account the user will be linked to.
+Optionally, the Account (and underlying Network and Instance) can be created in one request.
```

Comment:
Should this be public information? In other words, isn't this something only superadmins can do?